python/docs/source/reference/pyspark.sql/functions.rst

```diff
@@ -504,18 +496,6 @@ Generator Functions
     stack
 
 
-Partition Transformation Functions
-----------------------------------
-.. autosummary::
-    :toctree: api/
-
-    partitioning.years
```

Comment:
?? this is not deprecated.

Reply:
Ya, these are added recently.

Reply:
Ohhh,,, I was confused with `years`. Just revert the change. Thanks!!